integration/spark/src/test/common/java/io/openlineage/spark/agent/lifecycle/MatchesMapRecursively.java

```diff
@@ -32,24 +33,56 @@ public MatchesMapRecursively(Map<String, Object> target) {
         target);
   }
 
+  public static Predicate<List<Object>> predicate(List<Object> target, Set<String> omittedKeys) {
```

Comment:
There is description for class:
```
/**
 * Custom Condition writen for recursive comparison of Map and List with ability to ignore specified
 * Map keys AssertJ Built-in recursive comparison is not working with Map type, it can ignore only
 * object properties Example usage: assertThat(actualMap).satisfies(new
 * MatchesMapRecursively(expectedMap, new HashSet<>(Arrays.asList("runId"))));
 *
 * @see AbstractObjectAssert#usingRecursiveComparison()
 */
```